gcexport.py

```diff
@@ -88,63 +89,28 @@
 # Maximum number of activities you can request at once.
 # Used to be 100 and enforced by Garmin for older endpoints; for the current endpoint 'URL_GC_LIST'
 # the limit is not known (I have less than 1000 activities and could get them all in one go)
-LIMIT_MAXIMUM = 1000
+SINGLE_REQUEST_LIMIT = 1000
+# The absolute limit for activities is 10.000 on server side
+TOTAL_ACTIVITY_LIMIT = 10_000
```

Comment:
Can I suggest to revert the changes related to maximum number of activities? I tried using `--count all` in your branch and it tried to download only about 1000 activities, even though I have now more than 2000 activities (I know I should fix the comment on line 91 ;-))